index.html

```diff
@@ -37,65 +39,118 @@
     ga('send', 'pageview');
 
   </script>
+
   <link type="text/plain" rel="author" href="humans.txt" />
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400" rel="stylesheet">
 </head>
 
 <body>
-  <div class="ui container">
-    <div class="ui top fixed menu">
+<!--   <div class="ui container">
+    <div class="ui computer only secondary menu">
       <a class="item portfolio" href="index.html#portfolio"><span>portfolio</span></a>
       <a class="item values" href="index.html#values"><span>values</span></a>
       <a class="item team" href="index.html#team"><span>team</span></a>
       <a class="right item" href="contact.html"><span>contact</span></a>
     </div>
-  </div>
+  </div> -->
+  <div id="cover"></div>
   <div class="firstSegment segment">
-    <div class="mission-statement">
-      <span id="typing"></span>
-      <div class="statement">
-        <p>architect,</p>
-        <p>code,</p>
-        <p>deploy.</p>
-      </div>
+    <canvas id='background-grey-overlay'></canvas>
+    <canvas id="background-wtc"></canvas>
+    <div class="animated slideInDown mission-statement">
+      <h1>architect,</h1>
+      <h1>code,</h1>
+      <h1>deploy.</h1>
     </div>
-    <img class="logo" src="assets/Cortlandt-Labs-logo.png" />
-  </div>
-  <div class="secondSegment segment">
+    <img class="logo" src="assets/cortlandtlabs_logo_text.png" />
+    <i class="chevron circle down icon"></i>
   </div>
-  <div class="thirdSegment segment" id="portfolio">
-    <div class="ui container">
-      <h1 class="ui header"><i class="ui cortlandt-red small circle thin icon"></i>portfolio</h1>
+  <div class="secondSegment ui stackable padded grid ">
+    <div class="fourteen wide left aligned middle aligned centered column">
+      <h1>Are <span>YOU</span> looking to...</h1>
+      <h2>Design a <span>Proof of Concept</span> to raise capital?</h2>
+      <h2>Create a <span>Minimum Viable Product</span> to attract users?</h2>
+      <h2>Build a <span>Production-level App</span> to change the world?</h2>
+      <h1 id="partner">Partner with us</h1>
     </div>
-    <div class="ui stackable grid container">
-      <div class="seven wide column">
-        <div class="work1"></div>
-      </div>
-      <div class="seven wide column">
-        <div class="work2"></div>
-      </div>
-      <div class="seven wide column">
-        <div class="work3"></div>
-      </div>
-      <div class="seven wide column">
-        <div class="work4"></div>
+  </div>
+  <div class="thirdSegment ui padded grid">
+    <div class="ten wide middle aligned centered column">
+      <h1>Technology</h1>
+      <div class="ui divider"></div>
+      <div class="ui three middle aligned cards">
+        <div class="card">
+          <div class="image">
+            <img src="assets/rails-logo.png" />
```

Comment:
I think showing tech is that useful, but if we're going to put it, should we have a Ruby logo too?